src/uu/factor/BENCHMARKING.md

```diff
@@ -0,0 +1,113 @@
+# Benchmarking `factor`
+
+## Microbenchmarking deterministic functions
+
+We currently use [`criterion`] to benchmark deterministic functions,
+such as `gcd` and `table::factor`.
+
+Those benchmarks can be simply executed with `cargo bench` as usual,
+but may require a recent version of Rust, *i.e.* the project's minimum
+supported version of Rust does not apply to the benchmarks.
```

Comment:
@sylvestre Forgot to ask earlier: is this an OK decision to make?  Maintaining benchmarking code etc. for MinRustV could be pretty annoying.

Reply:
PS: I would have to move the benchmark to a separate crate (can't split the dependencies between test and bench otherwise) but that seems fine.

Reply:
OK per our IRC conversation